### DISCUSSION TITLE
Please check these

### DISCUSSION THREAD

**Discussion by venkat2305** [ReVanced/revanced-discussions#619] [Question] [answered]
1.Why Do we have to uninstall the previous version of revanced everytime we have to update the YouTube version? Can you fix this? 

2.Remove thanks and clip buttons under the player

3.Many people use the Newpipe x sponsorblock fork. Is it possible for the app to detect this Newpipe fork to download videos. This would be really great. 

4. When we update the patches we have to change the settings again and add sponsorblock Id and etc. Is it possible for the app to store the settings so we don't have to change settings every time?

**Comment by Fooftilly** [comment]
You don't have to uninstall Revanced to update it.

**Reply by oSumAtrIX** [reply, marked as answer]
No, the keystore allows to update without uninstalling
